src/main.jsx

```diff
@@ -93,10 +106,22 @@ document.addEventListener('DOMContentLoaded', () => {
     lang: data.cozyLocale
   })
 
+  let history = hashHistory
+  try {
+    var PiwikReactRouter = require('piwik-react-router')
+    const piwikTracker = (Piwik.getTracker(), PiwikReactRouter({
+      url: __PIWIK_TRACKER_URL__,
+      siteId: __PIWIK_SITEID__,
+      injectScript: false
+    }))
+    piwikTracker.push(['enableHeartBeatTimer'])
+    history = piwikTracker.connectToHistory(hashHistory)
+  } catch (err) {}
```

Comment:
Silent error ? Are we sure it's a good idea here ? At least a `console.warn()` could save a little developer mental health.